datasync/chaindatafetcher/api.go

```diff
@@ -39,8 +39,20 @@ func (api *PublicChainDataFetcherAPI) StopFetching() error {
 	return api.f.stopFetching()
 }
 
-func (api *PublicChainDataFetcherAPI) StartRangeFetching(start, end uint64, reqType uint) error {
-	return api.f.startRangeFetching(start, end, types.RequestType(reqType))
+func (api *PublicChainDataFetcherAPI) StartRangeFetching(start, end uint64, reqType interface{}) error {
```

Comment:
Is this okay to change the reqType?
It seems like it breaks compatibility.

Reply:
@kjhman21 Yes, it breaks compatibility. I think this is fine because the error message is understandable.

Or, I can make int type work too. How do you think of this?

Reply:
@jeongkyun-oh I'm not sure when this is implemented and how many people is using this, but it could be better to support the legacy unless we speak up this incompatible issue.

Reply:
So, what I mean is that we need to support previous usage. Can you make int type too?